pubspec.yaml

```diff
@@ -1,17 +1,23 @@
 name: html
-version: 0.15.0-dev
+version: 0.15.0-nullsafety-dev
 
 description: APIs for parsing and manipulating HTML content outside the browser.
 homepage: https://github.com/dart-lang/html
 
+publish_to: none
```

Comment:
add a comment here mentioning changing the version (removing -dev) before removing this line?